contrib/install/local.cfg

```diff
@@ -1,9 +1,6 @@
 PACKAGES="load:dealii-prepare once:cmake once:astyle once:hdf5 once:netcdf once:sundials once:p4est once:trilinos dealii"
 
-DEAL_II_VERSION=v9.5.2
```

Comment:
I think Rene did this on purpose because 9.5 is currently the minimum required and the version that is used by the official tester. This means there is value in having this version. @gassmoeller ?

Reply:
I mostly did it to make sure we do not have to deal with compile problems with different deal.II versions, but yes it makes sense to have the version at the same version we use for testing. Not sure if it is worth fixing right away, we could also wait until we require deal.II 9.6 and then update this file.